ci/ray_ci/doc/test_api.py

```diff
@@ -10,7 +11,9 @@
 )
 
 
-def test_from_autosummary():
+@mock.patch("ci.ray_ci.doc.api.API._fullname")
```

Comment:
mocking this to keep all data in this test file as fake; i tested the e2e integration of this function with autodoc in the `test_autodoc`